main.jsx

```diff
@@ -75,7 +75,8 @@ const Routes = () => (
     <Route exact path="/help" render={() => <Redirect to="/latest"/>} />
     <Route path="/help/:helpType" component={Help} />
     <Route path="/moderation" component={Moderation} />
-    <Route path="/profile/:id" component={Profile} />
+    <Route exact path="/profile/:id" component={Profile} />
+    <Route path="/profile/:id/:tab" component={Profile} />
```

Comment:
Doesn't React have a syntax for nesting in cases like this (would also be useful for `/help` above)?

Reply:
If i remember correctly in v4 it doesn't support the old way of nesting routes. I spent quite some time few months ago trying to get it working close to the old way but didn't find any solution. I can file a followup ticket to tackle this... don't want this to be the blocker.

Reply:
That's fine especially since `help` is already like this, was just a thought as I read through.